### PR TITLE
feat(lts): add host access resource

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
 + new resource **huaweicloud_lts_host_access**

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST="./huaweicloud/services/acceptance/lts" TESTARGS="-run TestAccHostAccessConfig"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lts -v -run TestAccHostAccessConfig -timeout 360m -parallel 4
=== RUN   TestAccHostAccessConfig_basic
=== PAUSE TestAccHostAccessConfig_basic
=== RUN   TestAccHostAccessConfig_windows
=== PAUSE TestAccHostAccessConfig_windows
=== CONT  TestAccHostAccessConfig_basic
=== CONT  TestAccHostAccessConfig_windows
--- PASS: TestAccHostAccessConfig_windows (16.74s)
--- PASS: TestAccHostAccessConfig_basic (23.24s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lts       23.285s
```

### CODE REVIEW

File: docs/resources/lts_host_access.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+* `paths` - (Required, List) Specifies the collection paths.
+
+  + A path must start with `/` or `Letter:\`.
+  + A path cannot contain only slashes (/). The following special characters are not allowed: <>'|"
+  + A path cannot start with `/**` or `/*`.
+  + Only one double asterisk (**) can be contained in a path.
+  + Up to 10 paths can be specified.
+
+* `black_paths` - (Optional, List) Specifies the collection path blocklist.
```
Comment (deer-hang) [inline docs/resources/lts_host_access.md]:
```suggestion
* `black_paths` - (Optional, List) Specifies the collection path blacklist.
```

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+}
+
+func testHostAccessConfig_basic(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+# A host group without hosts
+resource "huaweicloud_lts_host_group" "test" {
+  name     = "%[2]s"
+  type     = "linux"
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go]: Remove extra spaces and check the other parts.

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+resource "huaweicloud_lts_host_access" "test" {
+  name           = "%[2]s"
+  log_group_id   = huaweicloud_lts_group.test.id
+  log_stream_id  = huaweicloud_lts_stream.test.id
+  host_group_ids = [huaweicloud_lts_host_group.test.id]
+
+  access_config {
+    paths       = ["/var/log/*/*.log"]
+	black_paths = ["/var/log/*/a.log"]
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go]: Make fields alignment.

File: huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go [hunk 4]
```diff
@@ -0,0 +1,549 @@
...
+		return params
+	}
+	return nil
+}
+
+func buildSingleLogFormatRequestBody(rawParams map[string]interface{}) map[string]interface{} {
+	mode := rawParams["mode"].(string)
+	value := rawParams["value"].(string)
+
+	// get the current timestamp if valie is not specified in system mode
```
Comment (deer-hang) [inline huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go]:
```suggestion
	// get the current timestamp if value is not specified in system mode
```

File: huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go [hunk 5]
```diff
@@ -0,0 +1,549 @@
...
+
+func resourceHostAccessConfigUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	updateHostAccessConfigChanges := []string{
+		"access_config",
+		"host_group_ids",
+		"tags",
+	}
```
Comment (deer-hang) [inline huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go:459]: Only these three fields can be updated. I think `HasChanges` can be omitted.

Comment (ShiChangkuo) [reply huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go:459]: just keep consistent with the auto style.

File: huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go (hunk 4 above)
Comment (ShiChangkuo) [reply huaweicloud/services/lts/resource_huaweicloud_lts_host_access.go]: fixed

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go (hunk 3 above)
Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go]: fixed

File: huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go (hunk 2 above)
Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_host_access_test.go]: fixed

**Pull request merged**